protocol/avdl/keybase1/prove_common.avdl

```diff
@@ -86,4 +86,22 @@ protocol proveCommon {
     PGP_1002,
     ROOTER_100001
   }
+
+  record GenericServiceGroup {
+    // TODO need to spec out
+    string id;
+  }
+
+  record GenericServiceConfig {
+    boolean enabled;
+    boolean isDevel;
+    string displayName;
+    string domain;
+    union { null, GenericServiceGroup } group;
+    string usernameRe;
+    string prefillUrl;
+    string checkUrl;
+    array<string> checkPath;
+  }
```

Comment:
I feel like GenericServiceConfig since it's in `keybase1.` is a pretty abstract name since generic could be anything and service also means service. Maybe something like ProofServiceConfig, ProofServiceParams?

Reply:
yes better